.github/workflows/build-and-test.yml

```diff
@@ -33,7 +33,8 @@ jobs:
             %LocalAppData%\go-build
           key: v1-go-pkg-mod-${{ runner.os }}-${{ hashFiles('**/go.mod', '**/go.sum') }}
       - name: Run Unit tests
-        run: go test ./...
+        # use default Go build tags from Makefile.common
+        run: go test --tags containers_image_openpgp,exclude_graphdriver_btrfs,exclude_graphdriver_devicemapper ./...
```

Comment:
Do avoid duplication, perhaps turn this into a `makefile` target and call make from here.

Reply:
I think windows test job does not support makefile that's why the main command is duplicated as well here.